scripts/check_python

```diff
@@ -11,8 +11,10 @@ if hash "$PYTHON" 2>/dev/null; then
 	# Use full path from now on as it makes messages clearer
 	PYTHON=$(which $PYTHON)
 	PYTHON_VERSION=$($PYTHON -c 'import sys; print("%d.%d" % (sys.version_info[0], sys.version_info[1]))')
-	if [[ "$PYTHON_VERSION" != "2.7" ]]; then
-		echo "check-python: Python v2.7 not found (version reported by '$PYTHON' is $PYTHON_VERSION)." 2>&1
+	MINVER=2.7
+	MINVER3=3.4
+	if [ $(bc <<< "$PYTHON_VERSION < $MINVER || ($PYTHON_VERSION > 3.0 && $PYTHON_VERSION < $MINVER3)") -eq 1 ]; then
```

Comment:
@PaulPrice had the same comment. `bc` seems to be the sanest way to do floating point comparisons. I did find an example somewhere when searching for "bash floating point comparisons" but I have no idea where that page is now. I'll add a comment stating that it's for floats.